test/product-search-test.js

```diff
@@ -59,6 +59,98 @@ describe('Product Search', function() {
 
   });
 
+  it('summay should return appropriate products from store/s or brand/s 1', function(done){
```

Comment:
Typo in summary. This was also pointed out earlier on #11 [here](https://github.com/indix/indix-api-nodejs/pull/11#discussion_r107125189).